client/src/pages/Feed.js

```diff
@@ -258,6 +260,41 @@ const Feed = () => {
     dispatchAction(TOGGLE_STATE, "showFilters");
   }
 
+  const handlePostLike = (postId) => {
+    const { history, isAuthenticated, user } = props;
+
+    /* added here because userId not working */
+    sessionStorage.removeItem("likePost");
+
+    if (isAuthenticated) {
+      const endPoint = `/api/posts/${postId}/likes/${user && user.userId}`;
+      const post = posts.posts.filter((post) => post._id === postId);
```

Comment:
Maybe we could pass `post` as an argument instead of doing an extra search here?

Reply:
The PostSocial component is not receiving all the post object, I will do the change.

Reply:
Done.